build/common.ps1

```diff
@@ -240,8 +240,17 @@ Function Install-DotnetCLI {
 
         #If "-force" is specified, or folder with specific version doesn't exist, the download command will run"
         if ($Force -or -not (Test-Path $probeDotnetPath)) {
-            Trace-Log "$DotNetInstall -Channel $($cli.Channel) -InstallDir $($cli.Root) -Version $($cli.Version) -Architecture $arch -NoPath"
-            & $DotNetInstall -Channel $cli.Channel -InstallDir $cli.Root -Version $cli.Version -Architecture $arch -NoPath
+            $channelMainVersion = "5"
```

Comment:
We should be depending on 6.0 right now, so the channel probably needs to be interpreted different.

Reply:
We specify 5.0.2xx right now, but we should be using 6.0.1xx. So we can'treally hard code 5 here.

Reply:
Sure, it need to be future proof. Here `5` is just default placeholder value, but actual value is calculated from `$cli.Channel` variable and override it inside foreach loop (Line 244-251).  Also `$cli.Channel` itself is coming from build\config.props. 
`msbuild build\config.props /v:m /nologo /t:GetCliBranchForTesting` >> `release/5.0.2xx:5.0.200-servicing.21120.4`

Reply:
I see. 

Why should we set the placeholder value?
I'd argue we should fail if we can't fetch the correct version. 

wyt?

Reply:
Yes, we should fail if something goes wrong instead of being silent about it. Please check now.
https://github.com/NuGet/NuGet.Client/blob/ac9b29bc285c204b75b924befff5c91fc2bdb33e/build/common.ps1#L243-L255